### DISCUSSION TITLE
Video from ipywebrtc does not show in notebook.

### DISCUSSION THREAD

**Discussion by mapio** [microsoft/vscode-jupyter#9485] [General] [answered]
I'm trying to use [ipywebrtc](https://github.com/maartenbreddels/ipywebrtc) but when I run the example code from the documentation
```python
from ipywebrtc import CameraStream
CameraStream.facing_user()
```
I get `Error creating view for mediastream: Permission denied`.

Is this related to https://github.com/microsoft/vscode/issues/25172 there will be any possibility that this will be fixed?

**Comment by rchiodo** [comment, marked as answer]
That's actually this issue here:
https://github.com/microsoft/vscode/issues/118275

VS code doesn't ship with the necessary codecs to support video or audio. I'm not entirely sure why it's not supported (probably an OSS license thing)

So I think the answer is this won't be supported.
